CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue, with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a build.
+2. Update the README.md with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations and container parameters.
+
+## Code of Conduct
+
+### Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance and religion.
```

Comment:
On the phrase below I think it's better to change the 'and ' word to 'or' 
"...personal appearance and religion."

Reply:
I disagree.
We want a harassment-free experience regardless of all of those aspects, not one or another.

Reply:
I agree with you, it can be wrong understood with 'or' word, my bad

Reply:
What???

Reply:
It is resolved.